packages/inline-edit/src/styles/InlineEdit.css.ts

```diff
@@ -0,0 +1,50 @@
+import type { RecipeVariants } from '@vanilla-extract/recipes';
+
+import { vars } from '@launchpad-ui/vars';
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+const container = style({
+  display: 'flex',
+  gap: vars.spacing[300],
+  alignItems: 'center',
+});
+
+const inline = recipe({
+  variants: {
+    layout: {
+      vertical: { flexDirection: 'column', alignItems: 'flex-start' },
+      horizontal: { flexDirection: 'row' },
+    },
+  },
+});
+
+const cancelButton = style({
+  selectors: {
+    '.Button--icon&': {
+      height: '3rem',
+      width: '3rem',
+    },
+  },
+});
+
+const readButton = style({
+  display: 'inline-block',
+  padding: `${vars.spacing[200]} ${vars.spacing[300]}`,
+  borderRadius: vars.border.radius.regular,
+  ':hover': {
+    background: vars.color.bg.interactive.tertiary.hover,
+    cursor: 'pointer',
+  },
+  ':focus-visible': {
+    borderRadius: vars.border.radius.medium,
+    boxShadow: `0 0 0 2px ${vars.color.bg.ui.primary}, 0 0 0 4px ${vars.color.shadow.interactive.focus}`,
+    outline: 0,
+    zIndex: 2,
+  },
+});
+
+type InlineVariants = RecipeVariants<typeof inline>;
```

Comment:
This Vanilla Extract utility returns type `layout: "vertical" | "horizontal"` for the prop which means we can have a single source of truth for props tied directly to component styles (variants).